extensions/wikia/PhalanxII/hooks/PhalanxContentBlock.class.php

```diff
@@ -207,17 +207,17 @@ static public function editContent( $textbox, &$error_msg, $phalanxModel = null,
 	 *
 	 * hook
 	 */
-	static public function checkContent( $textbox, &$msg ) {
+	static public function checkContent( $textbox, &$msg, $displayBlock = false ) {
 		wfProfileIn( __METHOD__ );
 
 		$title = RequestContext::getMain()->getTitle();
 
 		$phalanxModel = new PhalanxContentModel( $title );
 
 		$ret = PhalanxContentBlock::editContent( $textbox, $msg, $phalanxModel );
-		
-		if ( $ret === false ) {
-			$msg = $phalanxModel->textBlock();
```

Comment:
Existing calls to `checkContent` used to call `textBlock` on this condition, but they'll totally skip it now. Is that intentional?